cdk/lib/support-workers.ts

```diff
@@ -0,0 +1,236 @@
+          maxAttempts: 999999, //If we don't provide a value here it defaults to 3
+          interval: Duration.seconds(1),
+          backoffRate: 2,
+        })
+        .addRetry({
+          errors: [Errors.ALL], // Wildcard to capture any error which originates from outside of our code (e.g. an exception from AWS)
+          maxAttempts: 999999,
+          interval: Duration.seconds(3),
+          backoffRate: 2,
+        });
+    };
+
+    const checkoutFailure = new Pass(this, "CheckoutFailure"); // This is a failed execution we don't want to alert on
+    const failState = new Fail(this, "FailState"); // This is a failed execution we do want to alert on
+
+    const succeedOrFailChoice = new Choice(this, "SucceedOrFailChoice")
+      .when(
+        Condition.booleanEquals("$.requestInfo.testUser", true),
+        checkoutFailure
+      )
+      .when(Condition.booleanEquals("$.requestInfo.failed", true), failState)
+      .otherwise(checkoutFailure);
+
+    const failureHandler = createLambda("FailureHandler", [
+      emailSqsPolicy,
+    ]).next(succeedOrFailChoice);
+
+    const catchProps = {
+      resultPath: "$.error",
+      errors: ["States.TaskFailed"],
+    };
+
+    const preparePaymentMethodForReuse = createLambda(
+      "PreparePaymentMethodForReuse"
+    ).addCatch(failureHandler, catchProps);
+
+    const createPaymentMethodLambda = createLambda(
+      "CreatePaymentMethod"
+    ).addCatch(failureHandler, catchProps);
+
+    const createSalesforceContactLambda = createLambda(
+      "CreateSalesforceContact"
+    ).addCatch(failureHandler, catchProps);
+
+    const createZuoraSubscription = createLambda("CreateZuoraSubscription", [
+      promotionsDynamoTablePolicy,
+    ]).addCatch(failureHandler, catchProps);
+
+    const sendThankYouEmail = createLambda("SendThankYouEmail", [
+      emailSqsPolicy,
+    ]);
+    const updateSupporterProductData = createLambda(
+      "UpdateSupporterProductData",
+      [supporterProductDataTablePolicy]
+    );
+    const sendAcquisitionEvent = createLambda("SendAcquisitionEvent", [
+      eventBusPolicy,
+    ]);
+
+    const shouldClonePaymentMethodChoice = new Choice(
+      this,
+      "ShouldClonePaymentMethodChoice"
+    );
+    const accountExists = Condition.booleanEquals(
+      "$.requestInfo.accountExists",
+      true
+    );
+    const checkoutSuccess = new Succeed(this, "CheckoutSuccess");
+
+    const parallelSteps = new Parallel(this, "Parallel")
+      .branch(sendThankYouEmail)
+      .branch(updateSupporterProductData)
+      .branch(sendAcquisitionEvent)
+      .branch(checkoutSuccess);
+
+    const commonSteps = createZuoraSubscription.next(parallelSteps);
+
+    const stepsWithExistingAccount =
+      preparePaymentMethodForReuse.next(commonSteps);
+
+    const stepsForNewAccount = createPaymentMethodLambda
+      .next(createSalesforceContactLambda)
+      .next(commonSteps);
+
+    const startPoint = shouldClonePaymentMethodChoice
```

Comment:
slightly more mathematic perhaps?
```suggestion
    const initialState = shouldClonePaymentMethodChoice
```